test/tests/rackhd11/test_rackhd11_api_config.py

```diff
@@ -44,14 +43,19 @@ def test_api_11_config_httpendpoints(self):
 
     def test_api_11_config_patch(self):
         api_data_save = fit_common.rackhdapi('/api/1.1/config')['json']
-        data_payload = {"CIDRNet": "127.0.0.1/22"}
+        if ("logColorEnable" in api_data_save and api_data_save['logColorEnable'] == True):
+            data_payload = {"logColorEnable": False}
+        else:
+            data_payload = {"logColorEnable": True}
         api_data = fit_common.rackhdapi("/api/1.1/config", action="patch", payload=data_payload)
         self.assertEqual(api_data['status'], 200, "Was expecting code 200. Got " + str(api_data['status']))
         for item in api_data['json']:
-            if fit_common.VERBOSITY >= 2:
-                print "Checking:", item
             self.assertNotEqual(item, '', 'Empty JSON Field:' + item)
```

Comment:
rather than just removing, change to a:
logs.debug("Checking: %s", item)

(you would need to all the following just below the 'import fit_common' line at the top of the file):

    import flogging
    
    logs = flogging.get_loggers()